src/ecrc-frontend/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "ecrc-frontend",
   "version": "0.1.0",
+  "homepage": "/criminalrecordcheck",
```

Comment:
does this work? 😮

Reply:
No, I think this must have been left in before I switched branches =( It doesn't work, and I broke everything

Reply:
Removed